### DISCUSSION TITLE
Monitoring (logging) which whitelisted urls are visited

### DISCUSSION THREAD

**Discussion by renaud** [SafeExamBrowser/seb-win-refactoring#842] [Q&A] [answered]
Bonjour,

I would like to whitelist an url (e.g. "https://ethercalc.net"), but monitor which exact urls are visited during the exam (e.g. https://ethercalc.net/94o9ivg88wr1). By monitor, I mean: get a complete list of all the urls as part of the exam submission.

Is this possible with SEB?

Thanks for your work on SEB, much appreciated.
Best, Renaud

**Comment by dbuechel** [comment, marked as answer]
Thanks for your feedback and the question. Yes indeed, in SEB for Windows, you can enable access to the real time application log, which in turn will reveal all requests executed by the internal browser engine and whether they're allowed or blocked:

1. Under tab Security, allow access to the application log and show the log button in the taskbar (see https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane).
2. Under tab Browser, then under restrictions in exam and additional window set to show URLs either always or before title (see https://safeexambrowser.org/windows/win_usermanual_en.html#BrowserPane).

With SEB for macOS, you'd even have a more convenient feature which allows you to teach SEB the allowed resp. blocked URLs while browsing in real time, see https://safeexambrowser.org/macosx/mac_usermanual_en.html#NetworkPane.
